### PR TITLE
tag assign on the basis of date and tag

### PR DESCRIPTION
**Pull request by vaibhavppathak**

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    "port": 8090,
+    "bodyLimit": "100kb",
+    "corsHeaders": ["Link"],
+    "db": {
+        "host": "localhost",
+        "name": "hr_system",
+        "password": "java@123",
+        "username": "root"
+    },
+    "CLIENT_ID": "26456531203-aa7qlroeerjrbaoatql4s5di1pcmir46.apps.googleusercontent.com",
```
Comment (mageexxcel) [inline config.json]: why is this new file created?

File: nano.save [hunk 2]
```diff
@@ -1 +0,0 @@
-Node send email to user on the basis of tag id with preconfigured template.
```
Comment (mageexxcel) [inline nano.save:1]: already told about this file to saurabh. this should not come in PR

File: src/inbox.js [hunk 3]
```diff
@@ -19,292 +15,258 @@ module.exports = {
...
-                                                    UID_arr = results;
-                                                } else {
-                                                    var row = resp[0];
-                                                    var Last_UID = row.get("uid");
-                                                    _.forEach(results, (val) => {
-                                                        if (val >= Last_UID) {
-                                                            UID_arr.push(val);
-                                                        }
-                                                    });
+                    let imap = new Imap({
```
Comment (mageexxcel) [inline src/inbox.js]: this wrong. there are services created for this.

File: src/inbox.js [hunk 4]
```diff
@@ -19,292 +15,258 @@ module.exports = {
...
-                                        let date = '';
-                                        if (err) {
-                                            console.log(err)
-                                        } else {
-                                            var row = resp[0];
-                                            if (row && row.get("email_date")) {
-                                                date = moment(new Date(row.get("email_date"))).format("MMM DD, YYYY");
-                                            } else {
-                                                date = moment(new Date()).format("MMM DD, YYYY");
+                    let imap = new Imap({
```
Comment (mageexxcel) [inline src/inbox.js]: wrong

File: src/modules/mail.js [hunk 5]
```diff
@@ -23,6 +23,7 @@ module.exports = {
                 if (error) {
                     reject("messsage not send successfully");
                 } else {
+                    console.log(response);
```
Comment (mageexxcel) [inline src/modules/mail.js]: remove

File: src/inbox.js [hunk 6]
```diff
@@ -4,11 +4,7 @@ import GENERIC from "./modules/generic";
 import _ from "lodash";
```
Comment (mageexxcel) [inline src/inbox.js]: wrong code in this file

File: src/modules/mail.js (hunk 5 above)
Comment (vaibhavppathak) [reply src/modules/mail.js]: done

**Pull request closed without merging**